<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakePreciseParametersControlWidget</class> 
 <widget class="QWidget" name="MakePreciseParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MakePreciseParameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>Should Make Precise</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="shouldMakePreciseCheckBox">
       <property name="toolTip">
         <string>Should Make Precise</string>
       </property>
       <property name="checked">
           <bool>0</bool>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>Algorithm</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="algorithmComboBox">
       <property name="toolTip">
         <string>Algorithm</string>
       </property>
       <property name="currentIndex">
         <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>KLT</string>
        </property>
       </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>Interpolation</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="interpolationComboBox">
       <property name="toolTip">
         <string>Interpolation</string>
       </property>
       <property name="currentIndex">
         <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Bilinear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spline</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polynom</string>
        </property>
       </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>KLT iterations</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="kLTIterationsSpinBox">
       <property name="toolTip">
         <string>KLT iterations</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>KLT Radius H</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="kLTRadiusHSpinBox">
       <property name="toolTip">
         <string>KLT Radius H</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
       <property name="text">
         <string>KLT Radius W</string>
       </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="kLTRadiusWSpinBox">
       <property name="toolTip">
         <string>KLT Radius W</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
       <property name="text">
         <string>KLT Threshold</string>
       </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="kLTThresholdSpinBox">
       <property name="toolTip">
         <string>KLT Threshold</string>
       </property>
       <property name="minimum">
        <double>1</double>
       </property>
       <property name="maximum">
        <double>100</double>
       </property>
       <property name="singleStep">
        <double>1</double>
       </property>
       <property name="value">
        <double>0</double>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>shouldMakePreciseCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MakePreciseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>algorithmComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MakePreciseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>interpolationComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MakePreciseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>kLTIterationsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MakePreciseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>kLTRadiusHSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MakePreciseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>kLTRadiusWSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MakePreciseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>kLTThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MakePreciseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
